### DISCUSSION TITLE
Change speed at layer not working?

### DISCUSSION THREAD

**Discussion by paulf123b** [jschuh/klipper-macros#163] [General]
Hi. Thanks for this awesome macro set.

This specific macro is not working for me; I've not tested any of the other during print ones yet.

The trigger happens but no change to actual print speed occurs. I manually issue the same command M220 S300 and it kicks in. 

Thanks for any help!

3:55 AM
layer 3:
* M220 S300
3:55 AM
speed_at_layer SPEED=300 LAYER=3

**Comment by jschuh** [comment]
It sounds like the [layer change gcode](https://github.com/jschuh/klipper-macros#before-layer-change-g-code) isn't configured properly for your slicer.
